app/views/layouts/_navigation.html.erb

```diff
@@ -60,9 +60,17 @@
     <li class="dropdown">
       <a href="#" class="dropdown-toggle" data-toggle="dropdown">
         Account
+        <% if user_signed_in? and session[:original_admin_user_id].present? %>
+          <span class="label label-warning"><%= current_user.username %></span>
```

Comment:
Just a note, the added width of the navigation bar to wrap into two lines on smaller resolutions when a lot of pending jobs/new events are shown (similar to #498). Since we don't know how long the user names I don't think we can do much about it.